docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,527 @@
+---
+sidebar_label: 'Read Profile Data'
+sidebar_position: 3.1
+---
+
+# Read Universal Profile Data
+
+In this guide, we will learn how to:
+
+- call a Universal Provile address
+- read data from a Universal Profile
+
+![Universal Profile example on universalprofile.cloud](./img/example-up.png)
+
+We will use:
+
+- [web3.js](https://web3js.readthedocs.io/en/v1.7.0/) for utility as well as connecting with the LUKSO network
+- [erc725.js](../../tools/erc725js/getting-started/) library to check the interface of an profile.
+- [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch) to enable you to use fetch in plain Node code
+
+## Table of Contents
```

Comment:
Maybe this table of content is not necessary, since it appears in the sidebar?

Otherwise, if you want to keep them, I would recommend adding the internal anchor links, like so in Markdown:

```markdown
1. [Check for valid Blockchain Address](#step-1---check-for-valid-blockchain-address)

```